tests/s3/test_copy_object.py

```diff
@@ -2257,3 +2261,809 @@ def test_22299(self):
             LOGGER.info(err.message)
             assert_utils.assert_in(errmsg.ACCESS_DENIED_ERR_KEY,err.message, err)
         LOGGER.info("ENDED: Use bucket policy to validate copy object with applied ACL.")
+
+    @pytest.mark.parallel
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-44750")
+    @CTFailOn(error_handler)
+    def test_44750(self):
+        """
+        Copy object Conditional params: All true for simple object copy"""
+        LOGGER.info(
+            "STARTED: Copy simple object with all conditional headers as true")
+        LOGGER.info("Step 1: Upload simple object to source bucket")
+        resp = system_utils.create_file(fpath=self.file_path, count=10, b_size="1M")
+        LOGGER.info(resp)
+        assert_utils.assert_true(resp[0], resp[1])
+        status, put_etag = self.create_bucket_put_object(self.s3_obj, self.bucket_name1,
+                                                         self.object_name1, self.file_path)
+        assert_utils.assert_true(status, put_etag)
+        LOGGER.info("Put object ETag: %s", put_etag)
+        obj_list = []
+        LOGGER.info("Step 2: Copy object to different bucket")
+        status, response = self.s3_obj.copy_object(self.bucket_name1, self.object_name1,
+                                                   self.bucket_name2, self.object_name2)
+        assert_utils.assert_true(status, response)
+        obj_list.append(self.object_name2)
+        LOGGER.info("Step 3: Copy object to different bucket with condition "
+                    "x-amz-copy-source-if-match (True)")
+        status, response = self.s3_obj.copy_object(self.bucket_name1, self.object_name1,
+                                                   self.bucket_name2, self.object_name2 + "_im",
+                CopySourceIfMatch=response[1]["ETag"])
```

Comment:
Please fix indentation in copy_object calls across the file, wherever applicable

Reply:
its available in latest commit